modules/op-pipelines-as-code-command-reference.adoc

```diff
@@ -0,0 +1,109 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/using-pipelines-as-code.adoc
+
+:_content-type: REFERENCE
+[id="pipelines-as-code-command-reference_{context}"]
+= {pac} command reference 
+
+[role="_abstract"]
+The `tkn-pac` CLI tool offers the following capabilities:
+
+* Bootstrap {pac} installation and configuration.
+* Create a new {pac} repository.
+* List all {pac} repositories.
+* Describe a {pac} repository and the associated runs.
+* Generate a simple pipeline run to get started.
+* Resolve a pipeline run as if it was executed by {pac}.
+
+== Basic syntax
+
+[source,terminal]
+----
+$ tkn pac [command or options] [arguments]
+----
+
+== Global options
+
+[source,terminal]
+----
+$ tkn pac --help
+----
+
+== Utility commands
+
+=== bootstrap
+
+.Bootstrapping {pac} installation and configuration
+[options="header"]
+|===
+
+| Command | Description 
+
+| `tkn pac bootstrap` | Installs and configures {pac} for Git repository hosting service providers, such as GitHub and GitHub Enterprise. 
+
+| `tkn pac bootstrap --nightly` | Installs the nightly build of {pac}.
+
+| `tkn pac bootstrap --route-url <public_url_to_ingress_spec>` | Overrides the OpenShift route URL. 
+
+By default, `tkn pac bootstrap` detects the OpenShift route, which is automatically associated with the {pac} controller service. 
+
+If you do not have an {product-title} cluster, it asks you for the public URL that points to the ingress endpoint.
+
+| `tkn pac bootstrap github-app` | Create a GitHub application and secrets in the `pipelines-as-code` namespace.
+
+|===
+
+=== repository
+
+.Managing {pac} repositories
+[options="header"]
+|===
+
+| Command | Description 
+
+| `tkn pac repo create` | Creates a new {pac} repository and a namespace based on the pipeline run template. 
+
+| `tkn pac repo list` | Lists all the {pac} repositories and displays the last status of the associated runs.
+
+| `tkn pac repo describe` | Describes a {pac} repository and the associated runs.
+
+|===
+
+=== generate
+
+.Generating pipeline runs using Pipelines as Code
+[options="header"]
+|===
+
+| Command | Description 
+
+| `tkn pac generate` | Generates a simple pipeline run. 
+
+When executed from the directory containing the source code, it automatically detects current Git information. 
+
+In addition, it uses basic language detection capability and adds extra tasks depending on the language. 
+
+For example, if it detects a `setup.py` file at the repository root, the link:https://hub.tekton.dev/tekton/task/pylint[pylint task] is automatically added to the generated pipeline run. 
+
+|===
+
+=== resolve
+
+.Resolving and executing pipeline runs using {pac}
```

Comment:
Needs more explanation like the comment before about why someone would use resolve

Reply:
@sm43, @chmouel - Please help me out here with some explanation, as asked by Siamak.

Reply:
@sounix000 we have description for resolve command  in docs
https://pipelinesascode.com/docs/guide/cli/#resolve
with a example
can we include that example and the description?